Dockerfile

```diff
@@ -15,6 +15,13 @@ ADD https://packages.microsoft.com/config/debian/12/packages-microsoft-prod.deb
 RUN  dpkg -i /tmp/packages-microsoft-prod.deb && \
   rm -rf /tmp/*
 
+# renovate: datasource=repology depName=debian_12/fontconfig versioning=deb
+ENV FONTCONFIG_VERSION=2.14.1-4
+
+RUN apt-get update -y && \
+  # Install necessary dependencies
+  apt-get install -y --no-install-recommends fontconfig=${FONTCONFIG_VERSION}
```

Comment:
Don't you need it in the final image?

Reply:
Oops, you are right.
Moved it to the final layer